src/app/board/pages/board-page/board-page.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Observable, Subscription } from 'rxjs';
+import { Location } from '@angular/common';
+import { ActivatedRoute } from '@angular/router';
+import { IColumn } from '../../models/column.model';
+import { IBoard } from '../../../main/models/board.model';
+import { ColumnApiService } from '../../services/column-api.service';
+import { BoardApiService } from '../../../main/services/board-api.service';
+import { MatDialog, MatDialogConfig } from '@angular/material/dialog';
+import { CreateColumnModalComponent } from '../../components/create-column-modal/create-column-modal.component';
+
+
+@Component({
+  selector: 'app-board-page',
+  templateUrl: './board-page.component.html',
+  styleUrls: ['./board-page.component.scss'],
+})
+export class BoardPageComponent implements OnInit, OnDestroy {
+  board: IBoard | undefined;
+
+  public columns$: Observable<IColumn[]> | undefined;
+
+  private routeParamsSub: Subscription | undefined;
+
+  private boardSub: Subscription | undefined;
```

Comment:
why undefined?

Reply:
oo, ok) 
it will be shorter in this case write so:
private routeParamsSub!: Subscription;